cmd/debos/debos.go

```diff
@@ -120,7 +139,7 @@ func main() {
 		}
 
 		log.Printf("==== Recipe done ====")
-		os.Exit(0)
+		panic(code0)
```

Comment:
I'm really not a fan of using `panic()` when there's no error.

Either we use a bare `return` here, relying on the fact that the process exitcode is `0` unless `os.Exit()` is called, or we [add a wrapper around `main()](https://github.com/anacrolix/torrent/blob/master/cmd/torrentfs/main.go#L72), which would probably also be a good place where to put the `recover()` deferred function.